src/client/components/robot_selector/robot_label/view.tsx

```diff
@@ -0,0 +1,76 @@
+import * as classNames from 'classnames'
+import { action } from 'mobx'
+import { observer } from 'mobx-react'
+import * as React from 'react'
+
+import { Collapsible } from '../../collapsible/view'
+import { RobotModel } from '../../robot/model'
+import { Switch } from '../../switch/view'
+import { StatusIndicator } from '../status_indicator/view'
+
+import DropdownIcon from './dropdown.svg'
+import { formatSI } from './format-si'
+import * as style from './style.css'
+import { RobotLabelViewModel } from './view_model'
+
+export type RobotLabelProps = {
+  robot: RobotModel
+  selectRobot(robot: RobotModel): void
+}
+
+export const RobotLabel = observer((props: RobotLabelProps) => {
+  const { robot, selectRobot } = props
+  const viewModel = RobotLabelViewModel.of(props.robot)
```

Comment:
I think I'm aiming to remove the concept of view models altogether, I really only want to use them to generate three.js scenes — maybe I should rename them to something else.

Anyway I think we can just use `RobotNetworkStatsModel.of(this.robotModel)` here 🙂